miniapps/tools/load-dc.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
Does the copyright for ``mfem`` files remain fixed at 2010?

Reply:
Yes